laser_encoders/laser_tokenizer.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/python3
```

Comment:
Can you please merge the [facebookresearch:MLH-dev](https://github.com/facebookresearch/LASER/tree/MLH-dev) branch back into this one?
This way, the changes in `laser_tokenizer.py` and `test_laser_tokenizer.py` that we have already reviewed will be offset, and the PR will display only the new changes that you introduce. 
It would make the PR much easier to review.